.github/workflows/CI.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+    pull_request:
+        branches: '**'
+
+jobs:
+    build:
+        runs-on: ${{matrix.os}}
+
+        strategy:
+            matrix:
+                node-version: [16.x]
```

Comment:
CI is being run on different node version than docker container, we should stick with only one